backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/BankAccountQuery.java

```diff
@@ -44,12 +48,14 @@
 
 	@Builder(toBuilder = true)
 	private BankAccountQuery(
-			final BPBankAcctUse bpBankAcctUse,
+			@Nullable @Singular final List<BPBankAcctUse> bpBankAcctUses,
 			@Nullable final BPartnerId bPartnerId,
 			@Nullable final InvoiceId invoiceId,
 			final Boolean containsQRIBAN)
 	{
-		this.bpBankAcctUse = bpBankAcctUse;
+		this.bpBankAcctUses = bpBankAcctUses != null ?
```

Comment:
IMHO u shall check if empty and set it to null....

Reply:
Done.